reactor-core/src/main/java/reactor/core/publisher/Operators.java

```diff
@@ -2307,10 +2324,18 @@ final void drainLoop() {
 
 		final T value;
 
+		final String stepName;
```

Comment:
this should probably be marked with the reactor `@Nullable` annotation on the line above it

Reply:
done